### DISCUSSION TITLE
Split screen

### DISCUSSION THREAD

**Discussion by ghost** [Divested-Mobile/DivestOS-Build#153] [Ideas] [answered]
In other ROM's I've tried there is an option to use split-screen by setting it for example to holding the recents key. I noticed that it doesn't work on DivestOS. There used to be an option before that didn't work but now even that doesn't seem to be there any more. Is there a specific reason?

**Comment by SkewedZeppelin** [comment, marked as answer]
Split screen works if you long press the app icon in the app switcher.

Also some devices running DivestOS are marked low_ram and don't have split screen available, among other features.

**Reply by ghost** [reply]
Cheers!
